srcpkgs/qgis/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'qgis'
+pkgname=qgis
+version=3.32.3
+revision=1
+archs="x86_64* i686"
+build_style=cmake
+configure_args="-DENABLE_TESTS=OFF -DWITH_3D=True -DWITH_PDAL=False
+ -DWITH_GRASS7=False -DWITH_GRASS8=True
```

Comment:
```suggestion
```
those aren't needed, they get set to those values automatically

Reply:
Removed